packages/fluent-theme/src/components/sendBox/SendBox.module.css

```diff
@@ -10,6 +10,13 @@
   --webchat-sendbox-border-radius: var(--webchat-borderRadiusLarge);
 }
 
+/* Copilot variant */
+:global(.webchat-fluent) .sendbox.variant-copilot {
```

Comment:
Do we have VRT for this?

Reply:
Not for the case when it overflows at least.